source/api.rst

```diff
@@ -0,0 +1,70 @@
+.. toctree::
+
+API
+---
+
+It's possbile to do all operations using the Onedata API.
+
+The official documentation is: https://onedata.org/#/home/api.
+
+.. important:: In order to be able to access the `Onedata APIs`, it's required
+    to generate an access token.
+
+Getting an API access token
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Tokens have to be generated from the `DataHub` (Onezone) interface as
+documented in :ref:`auth-token-using-web-interface` or using a command line
+call as documented hereafter.
+
+The following variables can be exported:
+
+* `OIDC_TOKEN`: OpenID Connect Access token, see
+  https://wiki.egi.eu/wiki/Federated_Cloud_OpenStack_Providers#Obtaining_an_access_token
+  for obtaining it.
```

Comment:
I think this should be Check-in documentation instead of Fedcloud openstack

Reply:
Changed the link to go to Check-in/AAI page.